newspaper/article.py

```diff
@@ -211,6 +212,14 @@ def parse(self):
             self.extractor.update_language(self.meta_lang)
             output_formatter.update_language(self.meta_lang)
 
+            # If there is no meta language provide, but detect_lang_from_title is true, and there is title text, try to detect language
+            if not self.meta_lang and self.config.detect_lang_from_title and self.title:
+                detected_lang = detect(self.title)
```

Comment:
I would rather use the more low-level API of getting probabilities instead of just picking the highest probability language.

For example:
what if the title was incredibly ambiguous and the "top lang detection" was only 10% sure? versus a long and obvious title that gets a 99% detection of English?

langdetect has a `get_probabilities(..)` API that allows fetching the direct probabilities. Let us grab the highest probability in 0th index and only use the detected language if it passes our threshold instead?
```
probs = langdetect.get_probabilities(self.title)
if len(probs) > 0 and probs[0].prob > SOME_THRESHOLD_PROBABLY_90_PERCENT:
  if probs[0].lang in get_available_languages():
      ... process
```
https://github.com/Mimino666/langdetect/blob/master/langdetect/detector.py#L141-L144

Does that work?